packages/decoupled/src/config/provide-config.ts

```diff
@@ -12,7 +12,7 @@ const configCache = new Map();
 /**
  * Read any config file match input pattern
  */
-function readConfigFromFiles(configDir: string, pattern: RegExp): AnyObject {
+export function readConfigFromFiles(configDir: string, pattern: RegExp): AnyObject {
```

Comment:
@friendken Those are actually private functions, which shouldn't be exposed to the public API. 
I'd suggest one of the two things: 
1. the cleanest solution would be to put them into separate modules, which are not part of the public API. That way you can test them internally, but don't need to expose them. 
2. there is something called `rewire` around like this: https://github.com/speedskater/babel-plugin-rewire i could be used for accessing internals for testing, but I'd opt for the cleaner solution from above unless there's no other way.